### DISCUSSION TITLE
Error while using local chromadb    TypeError: Can't instantiate abstract class MyVanna with abstract methods assistant_message, submit_prompt, system_message, user_message

### DISCUSSION THREAD

**Discussion by anuptibil** [vanna-ai/vanna#680] [Q&A]
<img width="1015" alt="Screenshot 2024-10-23 115529" src="https://github.com/user-attachments/assets/5a9834d1-768c-4c15-a256-6b409ed2dac0">

**Comment by zainhoda** [comment]
This configuration is invalid. If you select ChromaDB as your LLM you have to choose your own LLM. Vanna-hosted LLM only works with the Vanna-hosted vector database

**Reply by anuptibil** [reply]
oh okay, then some of these documentation and notebooks are misleading leading to these confusions
